tests/core/mempool/test_mempool_manager.py

```diff
@@ -84,10 +84,12 @@ def create_test_block_record(*, height: uint32 = TEST_HEIGHT) -> BlockRecord:
 
 
 async def instantiate_mempool_manager(
-    get_coin_record: Callable[[bytes32], Awaitable[Optional[CoinRecord]]]
+    get_coin_record: Callable[[bytes32], Awaitable[Optional[CoinRecord]]],
+    current_block_height: uint32 = TEST_HEIGHT,
+    current_block_timestamp: uint64 = TEST_TIMESTAMP,
```

Comment:
since both of these are integers, I think it would make sense to force them to be named as well

Reply:
Done.